docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,924 @@ The LDML specification is divided into the following parts:
+
+Here are some more examples. Note that if there is no order field or locale field in the PersonName object to be formatted, and the script of the PersonName data is different from that of the formatting locale, then the default result is givenFirst.
+
+| PersonName Object preferredOrder | PersonName Object Locale | Resulting Order |
+| -------------------------------- | ------------------------ | --------------- |
+| surnameFirst                     | ?                        | surnameFirst    |
+|                                  | zh                       | surnameFirst    |
+|                                  | und-JP                   | surnameFirst    |
+|                                  | fr                       | givenFirst      |
+|                                  |                          | givenFirst      |
+
+### 6.4 <a name="6-4-choose-a-personname" href="#6-4-choose-a-personname">Choose a personName</a>
+
+The personName data in CLDR provides representations for how names are to be formatted across the different axes of _order_, _length_, _usage_, and _formality_. More than one `namePattern` can be associated with a single `personName` entry. An algorithm is then used to choose the best `namePattern` to use.
+
+As an example for English, this may look like:
+
+```xml
+<personNames>
+    <personName order="givenFirst" length="long" usage="referring" formality="formal">
+<namePattern>{prefix} {given} {given2} {surname}, {suffix}</namePattern>
+    </personName>
+    <personName order="givenFirst" length="long" usage="referring" formality="informal">
+<namePattern>{given} «{given2}» {surname}</namePattern>
+<namePattern alt="2">«{given2}» {surname}</namePattern>
+    </personName>
+    <personName order="givenFirst" length="long" usage="sorting" formality="informal">
+<namePattern>{surname}, {given} {given2}</namePattern>
+    </personName>
+...
+</personNames>
+```
+
+The task is to find the best personName for a given set of input attributes. Well-formed data will always cover all possible combinations of the input parameters, so the algorithm is simple: traverse the list of person names until the first match is found, then return it.
+
+In more detail:
+
+A set of input parameters { order=O length=L usage=U formality=F } matches a personName element when:
+
+* The order attribute values contain O or there is no order attribute, and
+* The length attribute values contain L or there is no length attribute, and
+* The usage attribute values contain U or there is no usage attribute, and
+* The formality attribute values contain F or there is no formality attribute
+
+Example for input parameters 
+
+    order=**givenFirst** length=**long** usage=**referring** formality=**formal**
```

Comment:
At least in GitHub's Markdown view, the asterisks around `givenFirst`, `long`, `referring`, and `formal` are visible; they're not causing those words to show up in boldface.

Reply:
Yes, we found that the GFM format is sometimes fussy. Might have to have a space around the = sign.